src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="title">Moyo header</h1>
+    <header class="header">
+      <a href="/" class="img">
+        <img src="images/logo.png" alt="MOYO">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="menu apple"><a href="/" class="li is-active color">Apple</a></li>
```

Comment:
Need to fix code style and classes everywhere
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          
```